config/development.json

```diff
@@ -120,6 +120,7 @@
 		"layout/dotcom-nav-redesign": false,
 		"legal-updates-banner": true,
 		"livechat_solution": true,
+		"login/github": true,
```

Comment:
Minor / subjective comment: I tend to "pre-add" the feature flag to all five config files up-front.